src/Microsoft.AspNet.Mvc.Core/project.json

```diff
@@ -30,6 +30,10 @@
       "version": "1.0.0-*",
       "type": "build"
     },
+    "Microsoft.Extensions.HashCodeCombiner.Sources": {
+      "version": "1.0.0-*",
+      "type": "build"
+    },
```

Comment:
Nah. Remove this and put it back when it's used. What if you don't end up using it after we review part 3? Then this just sits here like a turd. :hankey: